mobile/test/java/org/chromium/net/impl/ErrorsTest.java

```diff
@@ -0,0 +1,109 @@
+package org.chromium.net.impl;
+
+import static org.chromium.net.impl.Errors.mapEnvoyMobileErrorToNetError;
+import static org.junit.Assert.assertEquals;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.SmallTest;
+import io.envoyproxy.envoymobile.engine.EnvoyFinalStreamIntelImpl;
+import io.envoyproxy.envoymobile.engine.UpstreamHttpProtocol;
+import org.junit.rules.ExpectedException;
+import org.chromium.net.impl.Errors.NetError;
+import org.junit.runner.RunWith;
+import org.junit.Rule;
+import org.junit.Test;
+
+@RunWith(AndroidJUnit4.class)
+public class ErrorsTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
```

Comment:
Is this needed? If an exception is thrown, the test will automatically file.

Reply:
I was just cargo culting, removed